Scripts/key.gd

```diff
@@ -14,3 +15,15 @@ func _process(delta):
 	
 func _physics_process(delta):
 	pass
+
+
+func _on_our_tile_grid_proportions(tile_scale, grid_size_in_tiles, tile_size):
+	
+	#first choses random x,y in tiles then converts that to position
+	var off_set = tile_size/2
+	var rand_pos_x = key_rng.randi_range(0,grid_size_in_tiles[0]) 
+	var rand_pos_y =  key_rng.randi_range(0,grid_size_in_tiles[1]) 
```

Comment:
I think we should use `randfn` here. 
'True' randomness can create frustrating repeated sequences, whereas normal distributions gives you an expectation of the kind games you'll have, and how often.

The primary consequence is that the corners of the screen will not be seen as often, but I think that's expected and desired. If not, it also **gives us control** over these things.

Reply:
Keeping it true random